crowdin.yml

```diff
@@ -0,0 +1,41 @@
+"project_id": "805338"
```

Comment:
@levieggertcru here's what the crowdin CLI config looks like, I think we will just end up using this to sync translations for android, iOS, & KMP projects

Reply:
I do still need to setup the GHA to push/pull translations using the CLI

Reply:
Thank you for tagging us.